packages/storybook/stories/components/code/documentation.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Meta, Markdown } from '@storybook/blocks';
+import SpecificationsCode from '@ovhcloud/ods-components/src/components/code/documentation/spec.md?raw';
+import { Banner } from '../../banner';
+import { DocNavigator } from '../../doc-navigator';
+import * as CodeStories from './code.stories';
+
+<Meta of={ CodeStories } name='Documentation' />
+
+<Banner of={ CodeStories } />
+
+# Overview
+
+Code component highlights strings or small blocks of code so it makes them easier to read and understand:
+
+<Canvas of={ CodeStories.Default } sourceState='none' />
+
+<DocNavigator of={ CodeStories } />
+
+<Markdown>{ SpecificationsCode }</Markdown>
+
+# Style customization
+
+You can update the container directly by adding your own class to it.
+
+<Canvas of={ CodeStories.CustomCSS } sourceState='shown'/>
+
+# Examples
+
+## Default
+
+<Canvas of={ CodeStories.Default } sourceState='shown' />
+
+## With copy button enabled
```

Comment:
To match with the other documentations I would maybe only put "CanCopy", or "Can Copy"